requirements.txt

```diff
@@ -28,6 +28,6 @@ pyyaml==3.*
 vulture==0.10.*
 bandit==1.1.*
 nbformat>=4.*
-pyflakes==1.2.*  # Although we don't need this directly, solves a dep conflict
+pyflakes==1.2.*
```

Comment:
The pyflakes line was removed from requirements.txt, so you now need to re-add it, as `~=1.3` please.